packages/plexus/src/Digraph/types.tsx

```diff
@@ -108,7 +108,7 @@ export type TMeasurableNodeRenderer<T = Record<string, unknown>> = {
 };
 
 export type TNodeRenderer<T = Record<string, unknown>> = {
-  renderNode: TRenderNodeFn<T>;
+  renderNode: TRenderNodeFn<T> | null;
```

Comment:
I see why this should be nullable, but it should be possible to still use this def in `NodesLayer.tsx` and `Nodes.tsx`, just making this one value [`NonNullable`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html)

Reply:
simplified `NodesLayer.tsx` and `Nodes.tsx` types